app-crypt/sops/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
```

Comment:
Non developers that want to maintain packages need to do it via proxy-maint as such they need to be a co-maintainer in metadatas.

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers